math/tan.go

```diff
@@ -0,0 +1,9 @@
+package math
+
+import (
+	"math"
+)
+
+func Tan(x float64) float64 {
+	return Cot((math.Pi / 2) - x)

```

Comment:
I don't think we have `Cot` defined, do we?

Anyways, we have `Sin` and `Cos` defined so why not use the `Sin/Cos`?